README.md

```diff
@@ -855,6 +855,7 @@ export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
 [[ -r $NVM_DIR/bash_completion ]] && \. $NVM_DIR/bash_completion
 ```
+Finally, use `nvm unload` to remove the nvm command from your terminal session.
```

Comment:
per the previous review, please move this to the beginning of the section. nvm should be unloaded before trying to uninstall it.

Reply:
Of course you are right. Almost done.